pairing (#75207)
Azure/azure-content: articles/governance/policy/concepts/guest-configuration.md

Customers designing a highly available solution should consider the redundancy planning requirements for
[virtual machines](../../../virtual-machines/availability.md) because guest assignments are extensions of
machine resources in Azure. When guest assignment resources are provisioned in to an Azure region that is

> [paired](../../../best-practices-availability-paired-regions), as long as at least one region in the pair

Reviewer: ```suggestion
[paired](../../../best-practices-availability-paired-regions.md), as long as at least one region in the pair
```

> is available then guest assignment reports will still be available. If the Azure region is not paired and

Reviewer: ```suggestion
is available, then guest assignment reports are available. If the Azure region isn't paired and
```

> it becomes unavailable, then it will not be possible to access reports for a guest assignment until

Reviewer: ```suggestion
it becomes unavailable, then it isn't possible to access reports for a guest assignment until
```
